primitives/ethereum-poa/src/lib.rs

```diff
@@ -549,7 +549,7 @@ pub fn step_validator<T>(header_validators: &[T], header_step: u64) -> &T {
 
 sp_api::decl_runtime_apis! {
 	/// API for querying information about headers from the Rialto Bridge Pallet
-	pub trait RialtoHeaderApi {
+	pub trait RialtoPoAHeaderApi {
```

Comment:
Luckily we don't have the bridge deployed anywhere :) BTW why do we hardcode the bridge name in the runtime API? This feels like it could be something more generic like: "PoAHeaderApi" - just curious, no changes required here.

Reply:
Because: (1) we have two instances of PoA bridge module in the runtime (2) API is implemented for runtime => we can't implement the same API twice => we need to include instance name into API.